### PR TITLE
Created specs

### PR DESCRIPTION
**Pull request by luizpicolo**
Run specs were created for the Run method

 - Created spec to test if the file was created;
 - Created spec to test if the file was not created;
 - Moved file to spec root;

### CODE REVIEW

File: lib/bun_bo.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@ def run(input_path)
       FileUtils.mkdir_p(test_folder)
       test_path.write('')
     else
-      puts "File does not exists"
+      "File does not exists"
```
Comment (hieuk09) [inline lib/bun_bo.rb]: If you returns the value here, you should also change the code in `exec/bun_bo`. Another solution is to raise an error here.

File: spec/bun_bo_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  before do
+    @input = "spec/fixtures/example.rb"
+    @output = "spec/fixtures/example_spec.rb"
+  end
+
+  describe "#run" do
+    it "should create a spec file" do
+      BunBo.new.run(@input)
+      expect(File.exist?(@output)).to be(true)
+      FileUtils.rm_rf(@output)
```
Comment (hieuk09) [inline spec/bun_bo_spec.rb]: You can use `File.delete` instead

**Review by hieuk09 — CHANGES_REQUESTED** [review]
Thank you for the pull request. Could you check my comments and amend the code?

File: lib/bun_bo.rb (hunk 1 above)
Comment (luizpicolo) [reply lib/bun_bo.rb]: Returned code to origin

File: spec/bun_bo_spec.rb (hunk 2 above)
Comment (luizpicolo) [reply spec/bun_bo_spec.rb]: Changed.

**Review by hieuk09 — APPROVED** [review]

**Comment by hieuk09** [comment]
❤️ ❤️ ❤️

**Pull request merged**